krkn/scenario_plugins/container/container_scenario_plugin.py

```diff
@@ -102,17 +103,37 @@ def container_killing_in_pod(self, cont_scenario, kubecli: KrknKubernetes):
                 # sys.exit(1)
                 raise RuntimeError()
             pods = pod_names
+        
+        # Get excluded pods if exclude_label is specified
+        exclude_pods = set()
```

Comment:
the list_pods function that is ussed to get the list of pods already accepts an exclude_label, let's use that instead of our own functionality. (note for some reason this wasn't working for me anyways)

https://github.com/krkn-chaos/krkn-lib/blob/3e7f7300451515f95e07ad212795e312e2e33388/src/krkn_lib/k8s/krkn_kubernetes.py#L532